docs/docs/glossary/react.md

```diff
@@ -0,0 +1,25 @@
+---
+title: React
+---
+
+Learn about React, including what it is, why you might use it, and how it fits into the Gatsby ecosystem.
+
+## What is React?
+
+React is a code library for building web-based user interfaces. It's written using [JavaScript](https://www.gatsbyjs.org/docs/glossary#javascript), one of the programming languages used to create web pages.
+
+Facebook first released React in 2013. The company still maintains the project, along with a community of contributors. It's free to use and open source under the terms of the [MIT License](https://github.com/facebook/react/blob/master/LICENSE).
+
+Where publishing tools such as WordPress and Jekyll rely on a system of template files to create a UI, React uses [components](/docs/glossary#component). Components are contained chunks of JavaScript, CSS, and HTML or SVG that can be reused, shared, and combined to create a web site or application.
+
+Components may be purely presentational. For example, you might create a `Logo` component that's just an SVG image. Or a component may encapsulate functionality. An `InputBox` component might include an input control, a label, and some simple validation.
+
+Components are also _composable_, which is a fancy way of saying that you can use multiple child components to create a parent component or view. Indeed, this is how to build Gatsby pages and templates.
```

Comment:
I'd like to rephrase "Indeed, this is how to build Gatsby pages and templates." Maybe "you will" or "you can" rather than "to"?

Reply:
Sounds good to me, as it makes it more of a way one could work than an absolute.